spec/models/post_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'should not create a post without title' do
+    post = Post.create(author: User.create(name: 'Aklilu', posts_counter: 0))
+    expect(post.valid?).to eq false
+  end
+
+  it 'should not create post with like/comment counter < 0' do
+    post = Post.create(author: User.create(name: 'Aklilu', posts_counter: 0), title: 'First Post',
+                       comments_counter: -1, likes_counter: -1)
+    expect(post.valid?).to eq false
+  end
+
+  it 'should not create post with like/comment counter not an integer' do
+    post = Post.create(author: User.create(name: 'Aklilu', posts_counter: 0), title: 'First Post',
+                       comments_counter: 'Hi', likes_counter: 'Hi')
+    expect(post.valid?).to eq false
+  end
+
+  it 'should create post with a title longer than 250 characters' do
```

Comment:
Kindly test for a scenario where the post length exceeds 250 characters. This is a part of the project requirements.